File: src/JS/1007.js
/*
 * 1007 - Difference
 * Author: Otavio
 * Date: 31/08/2024
 */
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const A = parseInt(lines.shift());
const B = parseInt(lines.shift());
const C = parseInt(lines.shift());
const D = parseInt(lines.shift());

console.log(`DIFERENCA = ${(A*B) - (C*D)}`)

File: src/C/1007.c
/*
 * 1007 - Difference
 * Author: Otavio
 * Date: 05/03/2024
 */
#include <stdio.h>

int
main()
{
    int a, b, c, d;
    
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    scanf("%d", &d);
    
    int diff = a*b-c*d;
    
    printf("DIFERENCA = %d\n", diff);
    return 0;
}
